dts/common/nordic/nrf54h20_enga.dtsi

```diff
@@ -243,28 +396,344 @@
+				max-frequency = <DT_FREQ_M(8)>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+			};
+
+			uart133: uart@9b6000 {
+				compatible = "nordic,nrf-uarte";
+				reg = <0x9b6000 0x1000>;
+				status = "disabled";
+				interrupts = <438 NRF_DEFAULT_IRQ_PRIORITY>;
+			};
+
+			timer134: timer@9c2000 {
+				compatible = "nordic,nrf-timer";
+				reg = <0x9c2000 0x1000>;
+				status = "disabled";
+				cc-num = <6>;
+				interrupts = <450 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-bit-width = <32>;
+				prescaler = <0>;
+			};
+
+			timer135: timer@9c3000 {
+				compatible = "nordic,nrf-timer";
+				reg = <0x9c3000 0x1000>;
+				status = "disabled";
+				cc-num = <6>;
+				interrupts = <451 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-bit-width = <32>;
+				prescaler = <0>;
+			};
+
+			i2c134: i2c@9c5000 {
+				compatible = "nordic,nrf-twim";
+				reg = <0x9c5000 0x1000>;
+				status = "disabled";
+				interrupts = <453 NRF_DEFAULT_IRQ_PRIORITY>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+			};
+
+			spi134: spi@9c5000 {
+				compatible = "nordic,nrf-spim";
+				reg = <0x9c5000 0x1000>;
+				status = "disabled";
+				easydma-maxcnt-bits = <15>;
+				interrupts = <453 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-frequency = <DT_FREQ_M(8)>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+			};
+
+			uart134: uart@9c5000 {
+				compatible = "nordic,nrf-uarte";
+				reg = <0x9c5000 0x1000>;
+				status = "disabled";
+				interrupts = <453 NRF_DEFAULT_IRQ_PRIORITY>;
+			};
+
+			i2c135: i2c@9c6000 {
+				compatible = "nordic,nrf-twim";
+				reg = <0x9c6000 0x1000>;
+				status = "disabled";
+				interrupts = <454 NRF_DEFAULT_IRQ_PRIORITY>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+			};
+
+			spi135: spi@9c6000 {
+				compatible = "nordic,nrf-spim";
+				reg = <0x9c6000 0x1000>;
+				status = "disabled";
+				easydma-maxcnt-bits = <15>;
+				interrupts = <454 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-frequency = <DT_FREQ_M(8)>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+			};
+
 			uart135: uart@9c6000 {
 				compatible = "nordic,nrf-uarte";
 				reg = <0x9c6000 0x1000>;
 				status = "disabled";
-				current-speed = <115200>;
 				interrupts = <454 NRF_DEFAULT_IRQ_PRIORITY>;
 			};
 
+			i2c136: i2c@9d5000 {
+				compatible = "nordic,nrf-twim";
+				reg = <0x9d5000 0x1000>;
+				status = "disabled";
+				interrupts = <469 NRF_DEFAULT_IRQ_PRIORITY>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+			};
+
+			spi136: spi@9d5000 {
+				compatible = "nordic,nrf-spim";
+				reg = <0x9d5000 0x1000>;
+				status = "disabled";
+				easydma-maxcnt-bits = <15>;
+				interrupts = <469 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-frequency = <DT_FREQ_M(8)>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+			};
+
 			uart136: uart@9d5000 {
 				compatible = "nordic,nrf-uarte";
 				reg = <0x9d5000 0x1000>;
 				status = "disabled";
-				current-speed = <115200>;
 				interrupts = <469 NRF_DEFAULT_IRQ_PRIORITY>;
 			};
+
+			timer136: timer@9d2000 {
```

Comment:
`timer136` and `timer137` should be located before `i2c136` to keep the address order.

Reply:
ditto